drake-torch.dockerfile

```diff
@@ -279,6 +282,13 @@ RUN apt-get update && apt-get install -y \
 RUN pip3 install --ignore-installed pyzmq
 RUN pip3 install jupyter
 
+# install nice-to-have some dev tools
+RUN apt-get -y update && apt-get -y upgrade && apt-get install -q -y \
+    clang-format \
```

Comment:
will this get the latest version?
clang-format 8 or 9?

Reply:
Looks like this installs `clang-format-6`

Reply:
I'll replace this with `clang-format-8`, then?

Reply:
Please update and move all to bottom.